src/display/canvas.js

```diff
@@ -705,8 +705,9 @@ var CanvasGraphics = (function CanvasGraphicsClosure() {
 
   CanvasGraphics.prototype = {
 
-    beginDrawing: function CanvasGraphics_beginDrawing(transform, viewport,
-                                                       transparency) {
+    beginDrawing: function CanvasGraphics_beginDrawing({
+      transform, viewport, transparency, backgroundColor = null,
+    }) {
```

Comment:
That seems to be the style of all the functions in the canvas.js file. I'm happy to clean that up as part of this pull request - but it might be better in a new one?